### DISCUSSION TITLE
Welcome to ElasticViews Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/ElasticViews#11] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jennifer066** [comment]
Hi, i have a question about this.
How can i add this implementation on android studio version 2.3.3? Because when i add this on the build.gradle its appers an error on another dependencies
Thanks:)

**Reply by skydoves** [reply]
Hi, @Jennifer066 
I can't remember the affordable version of that, try to import under the `2.0.1` versions. (1.0.0~2.0.1).

**Reply by Jennifer066** [reply]
@skydoves 
Ok, thanks
